.travis.yml

```diff
@@ -20,9 +20,9 @@ before_install:
   - touch local.properties
 
 script:
-  - ./gradlew clean assemble
+  - ./gradlew -Pcoverage --no-daemon clean assembledebug
```

Comment:
nit: While gradle accepts case insensitivity, I'd prefer to stick to their naming for scripts: -> `assembleDebug`